modules/ROOT/nav.adoc

```diff
@@ -88,11 +88,14 @@
 ***** xref:manage:kubernetes/storage/k-persistent-storage.adoc[PersistentVolume]
 ***** xref:manage:kubernetes/storage/k-hostpath.adoc[hostPath]
 ***** xref:manage:kubernetes/storage/k-emptydir.adoc[emptyDir]
-**** xref:manage:kubernetes/tiered-storage-kubernetes/index.adoc[Tiered Storage]
-***** xref:manage:kubernetes/storage/tiered-storage/k-tiered-storage.adoc[Tiered Storage]
-***** xref:manage:kubernetes/storage/tiered-storage/k-data-archiving.adoc[Data Archiving]
 **** xref:manage:kubernetes/storage/k-resize-persistentvolumes.adoc[Expand PersistentVolumes]
 **** xref:manage:kubernetes/storage/k-delete-persistentvolume.adoc[Delete PersistentVolumes]
+*** xref:manage:kubernetes/tiered-storage/index.adoc[Tiered Storage]
```

Comment:
It still feels a bit odd to have a section heading of Tiered Storage in the nav and then a duplicate sub-section heading underneath.

Reply:
Where's the duplicate?

Reply:
It's been updated. It now reads Tiered Storage > Use Tiered Storage.